libs/docusaurus/src/utils.ts

```diff
@@ -1,9 +1,12 @@
 import { BuilderContext } from '@angular-devkit/architect';
-import { join } from 'path';
+import { normalize, Path, resolve } from '@angular-devkit/core';
 
-export async function getProjectRoot(context: BuilderContext): Promise<string> {
+export async function getProjectRoot(context: BuilderContext): Promise<Path> {
   const projectMetadata = await context.getProjectMetadata(
     context.target.project
   );
-  return join(context.workspaceRoot, projectMetadata.root as string);
+  return resolve(
```

Comment:
This errors on Windows. Wrap with `getSystemPath`

Reply:
That's why `path` was switched to in the first place. 🤦  I didn't use `getSystemPath` here because the utils wouldn't be very well aligned if they're different.